setup.py

```diff
@@ -1,19 +1,49 @@
+#!/usr/bin/env python3
 """
 Packaging setup for Concierge CLI.
 See: https://click.palletsprojects.com/en/7.x/setuptools/
 """
-from setuptools import setup
+from os.path import abspath, dirname, join
+from setuptools import find_packages, setup
+
+import concierge_cli as package
+
+
+def read_file(filename):
+    """Get the contents of a file"""
+    here = abspath(dirname(__file__))
```

Comment:
why use a variable for this? it is not used anywhere else

Reply:
I'll have @basharSaid fix this in the packaging task. :+1: